contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,186 @@
+
+It was time for our bigger launch – [we posted on Hacker News](https://news.ycombinator.com/item?id=22376732), it went well (because we'd achieved the other steps first), and this got us well past this number.
+
+After the massive bump on launch day, however, our growth settled to a level that was noticeably higher than before.
+
+<Put in some kind of graph to show this>
+
+This phase is about figuring out what's working, and if you have a useful product you'll start to see a slow trickle of people recommending your tool, so you'll likely just get some growth without doing much marketing. We use [Syften](https://syften.com/) to keep an eye on this – it let's you spot people talking about your product online.
+
+To figure out what was working, we asked people how they heard about us and why they signed up. More specific still, we'd ask and track this question on every call we did, but we also asked people when they sign up via a text area – do *not* make this a selection, you want to see what people write unprompted. It's all too easy to assume Google is driving your traffic when people are searching for you because they heard about you elsewhere. It turned out that word of mouth growth was the majority. Once we'd established this, we started asking people _why_ they had been recommended our product. That turned out to be more helpful – _today_ that list looks like this:
+
+* All the tools in one
+  * to scale it - let's build more tools
+  * anti goal - spend all our money on sales and marketing so we can't have lots of developers
+* Pricing
+  * to scale it - let's cut pricing
+  * anti goal - don't ramp up prices just because people are happily paying already
+* Developer focus / brand
+  * to scale it - this is a longer piece to write!
+  * anti goal - don't stop taking risks with being outspoken online, for example
+* Support from technical people
+  * to scale it - add more technical people, improve tools and so on - again, we could write a lot about this
+  * anti goal - don't hire non technical people to scale support more cheaply
+
+Notice that the anti goals are all standard things that most companies follow, which is _exactly_ why bigger companies often become unpopular at scale.
+
+Haphazardly, we did experiment with one marketing channel – largely because I'm not good at coding, so figured I could help out this way. To do this, I first considered our users: engineers. I guessed they wouldn't appreciate being sold to, but that they would find following our journey interesting _and_ this was one topic I really could write about. So, I wrote about what my cofounder and I were up to – how we pivoted lots of times, or [what it was like going to San Francisco for YC](https://posthog.com/blog/moving-to-sf). This got us a couple more hits on the front page of Hacker News and thus we still do that to this day. So much so that to this day, now with over 60,000 companies using us, and I'm writing this blog post – as the CEO, at half midnight.
```

Comment:
```suggestion
At this stage, I experimented with content as a marketing channel – largely because I'm not good at coding, so figured I could help out this way. 

To do this, I first considered our users: engineers. I guessed they wouldn't appreciate being sold to, but that they would find following our journey interesting _and_ this was one topic I really could write about. So, I wrote about what my co-founder and I were up to – how we pivoted lots of times, or [what it was like going to San Francisco for YC](https://posthog.com/blog/moving-to-sf). 

This got us a couple more hits on the front page of Hacker News and thus we still do that to this day. So much so that to this day, now with over 60,000 companies using us, and I'm writing this blog post – as the CEO, at half midnight.

> #### What we learned
> 
> - Start tracking where your users hear about you. Ask them during calls, or at signup. _Do not_ rely on a dropdown list, give people a text box they can put anything in. It doesn't scale, but you'll get better answers.
>
> - Find out why people recommend you. This requires word-of-mouth growth, but understand _why_ is even more powerful than where or how people hear about you.
>
> - Use the above to nail down a list of goals and anti-goals based on your target market. These will be your guiding light while scaling your product, 
```